R/configuration-plan.R

```diff
@@ -263,8 +263,8 @@ ConfigurationPlan <- R6::R6Class(
       )
 
       reEnv$theme$fonts$legend$size <- self$plots$PlotSettings$Fonts$LegendSize %||% reEnv$theme$fonts$legend$size
-      reEnv$theme$fonts$xAxis$size <- self$plots$PlotSettings$Fonts$AxisSize %||% reEnv$theme$xAxis$legend$size
-      reEnv$theme$fonts$yAxis$size <- self$plots$PlotSettings$Fonts$AxisSize %||% reEnv$theme$yAxis$legend$size
+      reEnv$theme$fonts$xAxis$size <- self$plots$PlotSettings$Fonts$AxisSize %||% reEnv$theme$fonts$xAxis$size
+      reEnv$theme$fonts$yAxis$size <- self$plots$PlotSettings$Fonts$AxisSize %||% reEnv$theme$fonts$yAxis$size
       # TODO resizing of the input: normal size always appears bigger in background due to annotation_custom()
```

Comment:
@pchelle can you please verify that this change is ok, or was the legend fontsize being used for the xAxis fontsize?  Either way, `reEnv$theme$xAxis$legend$size` was returning `NULL`.